src/ui/public/vis/__tests__/_agg_config.js

```diff
@@ -189,6 +189,136 @@ describe('AggConfig', function () {
     });
   });
 
+  describe('#toJsonDataEquals', function () {
+    [{
+      type: 'metric',
+      aggs: [
+        {
+          type: 'count',
+          schema: 'metric',
+          params: {field: '@timestamp'}
+        }
+      ]
+    }, {
+      type: 'histogram',
+      aggs: [
+        {
+          type: 'avg',
+          schema: 'metric'
+        },
+        {
+          type: 'date_histogram',
+          schema: 'segment'
+        }
+      ]
+    }].forEach(function (visConfig, index) {
```

Comment:
I think we should use variable names here to increase readability, by assigning the array to a variable, and then calling forEach on the variable.

Not a big deal, but I think we should also default to using fat arrow functions if we're not naming the function.

```javascript
const visConfigs = [/* ... */];
visConfigs.forEach((visConfig, index) => {
  /* ... */
});
```